src/example-relay/src/Hotels/SSRQueryRenderer.js

```diff
@@ -5,7 +5,7 @@ import { QueryRenderer, type GraphQLTaggedNode } from '@adeira/relay';
 
 import createRelayEnvironment from '../createRelayEnvironment';
 
-type RendererProps = {| +[key: string]: any |};
+type RendererProps = any; // It should be any arbitrary *exact* object but how to annotate it?
```

Comment:
Relay also annotates this with `any`, see: https://github.com/facebook/relay/blob/master/packages/react-relay/ReactRelayQueryRenderer.js#L47